content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -76,6 +76,26 @@ For example, if a workflow defined the numOctocats and octocatEyeColor inputs, t
 
 **Optional** A `string` representing the default value. The default value is used when an input parameter isn't specified in a workflow file.
 
+### `inputs` for composite run steps actions
+
+Composite actions won't automatically convert an input argument into `INPUT_<VARIABLE_NAME>`, instead a similar behavior can be attained by manually mapping every variable wanted, for instance:
+
+```yaml
+inputs:
+  my-input-parameter:
+    description: 'An input parameter to illustrate the use case'
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - run: my-awesome-action.sh
+      shell: bash
+      env:
+        INPUT_MY_INPUT_PARAMETER: ${{ inputs.my-input-parameter }} 
```

Comment:
I don't see anything wrong with this line but the parameter is not displayed in deployed Heroku page. :S

![image](https://user-images.githubusercontent.com/10730372/109694641-cdd4f400-7b69-11eb-9eba-4d938adba990.png)

Anyone experienced could lend a hand here?

Reply:
👋 This parameter is being stripped out from rendering because during the docs build it is being interpreted as Liquid syntax.

You can get around this by escaping the code block with `{% raw %} ... {% endraw %}` tags. For example, this like this block: https://github.com/github/docs/pull/4151/files#diff-40c20329662b90a0f06aa88a004f251c8aed8a541325709b9e0058198c10cb4dL107-L120